disco_aws_automation/disco_group.py

```diff
@@ -35,17 +35,17 @@ def get_existing_group(self, hostclass=None, group_name=None, throw_on_two_group
             logger.info('Unable to get existing Spotinst group: %s', err.message)
             spot_group = []
         if asg_group and spot_group:
-            return sorted([asg_group.__dict__, spot_group], key=lambda grp: grp['name'], reverse=True)[0]
+            return sorted([asg_group, spot_group], key=lambda grp: grp['name'], reverse=True)[0]
         elif asg_group:
-            return asg_group.__dict__
+            return asg_group
         elif spot_group:
             return spot_group
         else:
             logger.info('No group found')
 
     def get_existing_groups(self, hostclass=None, group_name=None):
         asg_groups = self.autoscale.get_existing_groups()
-        asg_groups = [group.__dict__ for group in asg_groups]
+        asg_groups = [group for group in asg_groups]
```

Comment:
I believe this is not needed anymore. `self.autoscale.get_existing_groups()` returns a list.